tox.ini

```diff
@@ -47,20 +47,12 @@ usedevelop = False
 commands = bandit-baseline -r bandit -ll -ii
 
 [testenv:cover]
-setenv =
-    {[testenv]setenv}
-    PYTHON=coverage run --source bandit --parallel-mode
+sitepackages = False
 commands =
-    coverage erase
-    stestr run '{posargs}'
-    coverage report
-
-[testenv:openstack_coverage]
-deps =
- -c{env:UPPER_CONSTRAINTS_FILE:https://git.openstack.org/cgit/openstack/requirements/plain/upper-constraints.txt}
- PyYAML>=3.1.0
-       requests>=2.7.0
-commands = python tools/openstack_coverage.py
+  coverage erase
+  coverage run --source bandit -m testtools.run discover tests
```

Comment:
If you're using codecov you'll want to aggregate coverage across all versions of python and let codecov aggregate them.

We configure it here: https://github.com/PyCQA/flake8/blob/master/codecov.yml

And we use GitLab CI to upload here: https://github.com/PyCQA/flake8/blob/33e7eae53ad513952da777cabd3654219efe0eb8/.gitlab-ci.yml#L15

I think that would work well here